lib/cli/test/fixtures/meteor/package.json

```diff
@@ -1,12 +0,0 @@
-{
-  "name": "meteor-fixture",
-  "version": "1.0.0",
-  "private": true,
-  "scripts": {
-    "start": "meteor run"
-  },
-  "dependencies": {
-    "@babel/runtime": "7.0.0",
-    "meteor-node-stubs": "~0.2.4"
-  }
-}
```

Comment:
It was simply not working at all anymore...

The build was failing to find `/dist`, perhaps it can be saved with proper investigation. I'm unsure if it's worth the time.